target/linux/generic/backport-5.4/000-rk3328-friendlyarm-apply-on-linux-5-4-40.patch

```diff
@@ -0,0 +1,9506 @@
+diff -ruN linux-5.4.40/Documentation/devicetree/bindings/phy/phy-rockchip-inno-usb3.txt kernel-rockchip/Documentation/devicetree/bindings/phy/phy-rockchip-inno-usb3.txt
```

Comment:
This seems to be mostly unrelated (it adds additional devices?), despite it doesn't belong into generic?